File: Programming Basics with Python/По-сложни проверки/hotel_room.py

month = input()
overnight_stays = int(input())

if month == "May" or month == "October":
    studio = 50
    apartment = 65
    if 7 < overnight_stays < 14:
        studio *= 0.95
    elif overnight_stays > 14:
        studio *= 0.70
elif month == "June" or month == "September ":
    studio = 75.20
    apartment = 68.70
    if overnight_stays > 14:
        studio *= 0.80
elif month == "July" or month == "August":
    studio = 76
    apartment = 77


if overnight_stays > 14:
        apartment *= 0.90

studio_price = studio * overnight_stays
apartment_price = apartment * overnight_stays

print(f"Apartment: {apartment_price:.2f} lv.")
print(f"Studio: {studio_price:.2f} lv.")

File: Programming Basics with Java/More complex checks/HotelRoom.java
import java.util.Scanner;

public class HotelRoom 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);

        String month = input.nextLine();
        int days_spend = Integer.parseInt(input.nextLine());

        double studio = 0.0;
        double apartment = 0.0;
        double discount_studio = 0.0;
        double discount_apartment = 0.0;

        switch (month)
        {
            case "May":
            case "October":
            {
                if (days_spend > 14)
                {
                    discount_studio = 0.7;
                }
                else if (days_spend > 7)
                {
                    discount_studio = 0.95;
                }
                studio = 50;
                apartment = 65;
                break;
            }
            case "June":
            case "September":
            {
                if (days_spend > 14)
                {
                    discount_studio = 0.8;
                }
                studio = 75.2;
                apartment = 68.7;
                break;
            }
            case "July":
            case "August":
            {
                studio = 76;
                apartment = 77;
                break;
            }
        }

        if (discount_studio != 0.0)
        {
            studio *= discount_studio;
        }

        if (days_spend > 14)
        {
            apartment *= 0.9;
        }

        System.out.printf("Apartment: %.2f lv.\n", apartment * days_spend);
        System.out.printf("Studio: %.2f lv.", studio * days_spend);
    }
}
